flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/parameters/KubernetesJobManagerParameters.java

```diff
@@ -64,10 +63,15 @@ public KubernetesJobManagerParameters(
                         .getOptional(KubernetesConfigOptions.JOB_MANAGER_LABELS)
                         .orElse(Collections.emptyMap()));
         labels.putAll(getCommonLabels());
```

Comment:
@Aitozi We have put the common labels twice. Right?

Reply:
@wangyang0918  Yes, But I think the selector do not always rely on the common labels. So I do not remove this line. 
So that we do not have to care about the details of selector and common labels (although the selector contains the common labels, now), We just have to make them up to build the jobManager and taskManager labels. It's my consideration.

Reply:
The code here only describes that: `user label + common label + selector` to build up the pod label .

Reply:
Currently, the selector has already contained the common labels. So I do not think `user label + common label + selector` makes sense to me.

BTW, `KubernetesTaskManagerParameters#getLabels` has different semantic. `Labels = user label + selector(internal labels)`.

Reply:
Got it, I can remove this duplicate common labels, I will push the commit .